### ISSUE TITLE
Examples of false positives

### ISSUE THREAD

**Issue by upekkha** [rvben/rumdl#186]
Thanks for this very useful project.

I stumbled across the following false positives.

````markdown
Setext Header with colon : inside
=================================

* List item with asterisk * inside

* Some list
  * item1

  ```sh
  echo "code"
  ```

  * item2
    * correctly indented subitem
````

```
1:1: [MD041] First line in file should be a level 1 heading
4:1: [MD037] Spaces inside emphasis markers: "* List item with asterisk *" [*]
14:1: [MD005] Expected indentation of 0 spaces, found 4 [*]
```

**Comment by rvben** [comment]
You found some bugs, @upekkha! Thanks for letting me know. These should all be resolved in 0.0.192. If you hit anything else, let me know! 🚀
